packages/elements/src/components/ino-range/ino-range.scss

```diff
@@ -1,83 +1,69 @@
 @use 'base/mdc-customize';
 @use '@material/slider/slider';
 @use '@material/slider' as slider-mixins;
-@use 'base/theme';
+@use 'base/new-theme' as *;
```

Comment:
It would be nice to have an alias (e.g. `theme`) for this import and then call the colors like this: `theme.$primary`